addons/html_builder/static/src/utils/grid_layout_utils.js

```diff
@@ -189,8 +189,8 @@ function placeColumns(columnEls, rowSize, rowGap, columnSize, columnGap, mobileB
         const imageEl = columnEl.querySelector("img");
         // Checking if the column has a background color to take that into
         // account when computing its size and padding (to make it look good).
-        const hasBackgroundColor = columnEl.classList.contains("o_cc");
-        const isImageWithoutPadding = isImageColumn && !hasBackgroundColor;
+        const hasBackground = columnEl.matches(".o_cc, .oe_img_bg");
```

Comment:
Nitpick but you didn't change the comment here, let's keep the commits consistent :wink: 
=> `has a background color ` -> `has a background color/image`